src/crc32c_arm64.cc

```diff
@@ -88,7 +90,9 @@ uint32_t ExtendArm64(uint32_t crc, const uint8_t *data, size_t size) {
     t2 = (uint64_t)vmull_p64(crc2, k2);
     t1 = (uint64_t)vmull_p64(crc1, k1);
     t0 = (uint64_t)vmull_p64(crc0, k0);
-    crc = __crc32cd(crc3, *(uint64_t *)data);
+    uint64_t d;
```

Comment:
Nit: These temporaries could all be declared at the top of the function outside of the loops. I honestly don't know if any modern compiler cares. And the benchmarks are unaffected by the way they're done here, so I don't have a strong opinion.

Reply:
Thanks! Updated.